crates/sui-core/src/authority/authority_store.rs

```diff
@@ -1502,6 +1502,13 @@ impl<S: Eq + Serialize + for<'de> Deserialize<'de>> SuiDataStore<S> {
     pub fn get_latest_authenticated_epoch(&self) -> Option<AuthenticatedEpoch> {
         self.epochs.iter().skip_to_last().next().map(|(_, a)| a)
     }
+
+    pub fn get_sui_system_state_object_ref(&self) -> SuiResult<ObjectRef> {
+        let sui_system_object = self
+            .get_object(&SUI_SYSTEM_STATE_OBJECT_ID)?
+            .expect("Sui System State object must always exist");
```

Comment:
There should already be a `get_sui_system_state_object`?

Reply:
We might need the object_ref for other programatic move calls, but I'll remove it for now.